api/src/main/java/org/jenkinsci/plugins/workflow/graph/FlowGraphWalker.java

```diff
@@ -53,4 +56,59 @@ public FlowNode next() {
         }
         return null;
     }
+
+    @Override
+    public Iterator<FlowNode> iterator() {
```

Comment:
:ant: document that, unlike `Iterable.iterator` generally, this may be iterated only once.